.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -133,8 +105,8 @@ fi
 ################################################################################################################
 # VALIDATE DATABASE SUPERUSERS
 
-if [ "${WORKER}" == 0 ]; then
-    DATABASE_FILE="$CONF_DIR/database.json"
+if [ "${IS_WORKER}" == 0 ]; then
+    DATABASE_FILE="$SUBMITTY_INSTALL_DIR/config/database.json"
```

Comment:
Is this different from $SUBMITTY_CONFIG_DIR?

Reply:
No, they are the same. In this script I avoided using `SUBMITTY_CONFIG_DIR` so that we can remove that variable in the future with another refactor.